packages/css-tokenizer/LICENSE.md

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
MIT Licensed because it is intended as a low level package.
This matches what PostCSS uses and should be more compatible with the ecosystem.